tests/integration/test_api.py

```diff
@@ -470,3 +477,45 @@ def test_annotate_insight_anonymous_then_authenticated(client, mocker):
         server_domain=settings.OFF_SERVER_DOMAIN,
         auth=OFFAuthentication(username="a", password="b"),
     )
+
+
+def test_get_images(client):
```

Comment:
I forgot to do it in last PR but we should separate test of the API and test of the function.

Test on the function should be in test_core.py.

On the API you should try one empty test, but also a test with some data.